src/stores/GroupStore.js

```diff
@@ -96,6 +105,10 @@ export default class GroupStore extends EventEmitter {
         this.removeListener('update', fn);
     }
 
+    isStateReady(id) {
```

Comment:
This is quite nice but I feel like we now need to doc what string values are acceptable to it. How about defining some constants on the GroupStore class and and using these instead?